packages/daimo-common/src/foreignCoin.ts

```diff
@@ -1,4 +1,5 @@
-import { Address, formatUnits } from "viem";
+import { WETH9 } from "@uniswap/sdk-core";
```

Comment:
I think you could drop all the changes in this file for now and keep them for #1124